File: app/src/main/java/com/stablekernel/interview/api/model/TokenResponse.java
package com.stablekernel.interview.api.model;

import com.google.gson.annotations.SerializedName;

public final class TokenResponse {

    @SerializedName("access_token")
    private String accessToken;

    public String getAccessToken() {
        return accessToken;
    }

    @Override
    public String toString() {
        final StringBuilder sb = new StringBuilder("TokenResponse{");
        sb.append("accessToken='").append(accessToken).append('\'');
        sb.append('}');
        return sb.toString();
    }

    public String getBearerToken() {
        return new StringBuilder("Bearer ").append(accessToken).toString();
    }
}

File: app/src/main/java/com/stablekernel/interview/login/api/model/TokenResponse.kt
package com.stablekernel.interview.login.api.model

import com.google.gson.annotations.SerializedName

class TokenResponse(@SerializedName("access_token") val accessToken: String) {

    val bearerToken = "Bearer $accessToken"

    override fun toString() = "TokenResponse{accessToken='$accessToken'}"
}
